### PR TITLE
Upgrade v12 actors to v12.0.0-rc.3

### PR DESCRIPTION
**Pull request by sudo-shashank**
**Summary of changes**
Changes introduced in this pull request:
- Updated v12 actors to latest release of builtin actors v12.0.0-rc.3
https://github.com/filecoin-project/builtin-actors/releases/tag/v12.0.0-rc.3
rc2 -> rc3 : https://github.com/filecoin-project/builtin-actors/compare/v12.0.0-rc.2...v12.0.0-rc.3
- Updated known and manifest cids
- Added list of possible miners for calibnet

**Reference issue to close (if applicable)**

Contributes to #203 

**Other information and links**

### CODE REVIEW

**Review by hanabi1224 — APPROVED** [review]

**Comment by LesnyRumcajs** [comment]
@sudo-shashank I believe we need to add the previous miner to possible miners as an exception, sadly, otherwise Forest will fail on some past parts of the calibration network.

see https://github.com/ChainSafe/fil-actor-states/pull/196

**Comment by sudo-shashank** [comment]
> @sudo-shashank I believe we need to add the previous miner to possible miners as an exception, sadly, otherwise Forest will fail on some past parts of the calibration network.
> 
> see #196

fixed

**Comment by LesnyRumcajs** [comment]
@sudo-shashank given that we want to test it on the devnet, we're going to need to add all previous NV21 miner cids (basically from all v12 RCs) for devnet as well. Could. you please add them?

**Comment by sudo-shashank** [comment]
> @sudo-shashank given that we want to test it on the devnet, we're going to need to add all previous NV21 miner cids (basically from all v12 RCs) for devnet as well. Could you please add them?

got it, i'll add

File: fil_actor_interface/src/builtin/miner/mod.rs [hunk 1]
```diff
@@ -43,12 +42,25 @@ pub fn is_v11_miner_cid(cid: &Cid) -> bool {
 }
 
 pub fn is_v12_miner_cid(cid: &Cid) -> bool {
-    // The following code cid existed temporarily on the calibnet testnet, as a "buggy" storage miner actor implementation.
+    // The following code cid's existed temporarily on the calibnet testnet, as a "buggy" storage miner actor implementation.
```
Comment (LesnyRumcajs) [inline fil_actor_interface/src/builtin/miner/mod.rs:45]: This should be plural

**Review by LesnyRumcajs — APPROVED** [review]

**Review by hanabi1224 — APPROVED** [review]

**Pull request merged**